unlock-app/src/contexts/AuthenticationContext.tsx

```diff
@@ -23,13 +25,37 @@ export const defaultValues = {
   watchAsset: () => {},
   providerSend: (_method: string, _params: string[]) => {},
   getWalletService: async (_network?: number) => new WalletService(networks),
+  accessToken: null,
 }
 
 export const AuthenticationContext =
   createContext<AuthenticationContextType>(defaultValues)
 
 export const useAuth = () => {
-  return useContext(AuthenticationContext)
+  const context = useContext(AuthenticationContext)
```

Comment:
mapping privy auth states to our auth context. adding `accessToken` deems unnecessary. will remove.